packages/api/src/command/medical/admin/upload-doc.ts

```diff
@@ -61,7 +67,7 @@ export async function createAndUploadDocReference({
       {
         resourceType: "Organization",
         id: orgRef,
-        name: metadata.orgName ?? `Hospital ${orgRef}`,
+        name: metadata.organizationName ?? `Hospital ${orgRef}`,
```

Comment:
Like we talked on the call, this whole function needs reviewing so it uses customer data